packages/external-db-config/lib/readers/common_config_reader.js

```diff
@@ -4,8 +4,8 @@ class CommonConfigReader {
     constructor() { }
 
     readConfig() {
-        const { CLOUD_VENDOR, TYPE, REGION, SECRET_NAME } = process.env
-        return { vendor: CLOUD_VENDOR, type: TYPE, region: REGION, secretId: SECRET_NAME }
+        const { CLOUD_VENDOR, AUTH_VENDOR, TYPE, REGION, SECRET_NAME } = process.env
+        return { vendor: CLOUD_VENDOR, authVendor: AUTH_VENDOR, type: TYPE, region: REGION, secretId: SECRET_NAME }
```

Comment:
@noam-almog  not sure if we need to verify `AUTH_VENDOR` variable (check that it was supplied) 
Will there always be authorization?
WDYT?

Reply:
auth vendor is the same as vendor, no ?

Reply:
We agreed that these would be 2 different things …

Reply:
why ? it's the same value ? aws/gcp/azure, no ?